examples/tunnel/tunnel.go

```diff
@@ -0,0 +1,116 @@
+// main is an example of tunneling traffic over grpc
```

Comment:
Any reason to have this example lying around once we implement the CLI version? Or, do we expect users to write their own programmatic Golang code creating their own tunnels?

Reply:
good point, I'll remove it during the CLI task. In theory, users could write a tunneling program using a passed in connection/rwcloser, but the CLI code should suffice as an example